apps/cuda_mat_mul/runner.cpp

```diff
@@ -1,3 +1,8 @@
+// We always want asserts
+#ifndef NDEBUG
```

Comment:
Shouldn't this be `ifdef` rather than `ifndef`

Reply:
Oops. This is a bug in the underlying branch so I'll fix it there.